src/samui/frontend/src/routes/Dashboard/Violations.tsx

```diff
@@ -65,7 +66,21 @@ class Alerts extends React.PureComponent<AlertsProps> {
               !currentRule ? (
                 'Violations Dashboard'
               ) : currentRule.savedBody ? (
-                currentRule.title
+                <div>
+                  <Input
+                    id="title_input"
+                    style={{width: 300}}
+                    defaultValue={currentRule.title}
+                    onChange={e => (currentRule.newTitle = e.target.value)}
```

Comment:
ditto — pass it through via an action and a reducer into the redux state, then it will re-render. I know it's a little tedious to add this for simply editing a textbox, and personally prefer how e.g. Angular does two-way binding, but alas this is what we've gone with for now.

Reply:
This has been addressed!